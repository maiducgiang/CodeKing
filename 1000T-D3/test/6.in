6
7690 9874 6958 7667
3770 5141 5756 7827
6365 1810 1438 4667
7078 3288 1127 5537
1605 8021 4448 5222
120 337 3090 4280
