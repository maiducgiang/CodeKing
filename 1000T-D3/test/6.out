74155502
37399266
11755650
11682482
36919678
1184130
